### DISCUSSION TITLE
[V6] Assets loading errors - not found - css error - failed to load

### DISCUSSION THREAD

**Discussion by christmex** [Laravel-Backpack/community-forum#512] [Tips & Tricks]
Hi guys, I just installed Backpack v6, and I got this error, both in the login page and dashboard page.
<img width="573" alt="image" src="https://github.com/Laravel-Backpack/community-forum/assets/34405837/7b7983a2-93da-44dc-8c73-4e763d902a2c">
<img width="926" alt="image" src="https://github.com/Laravel-Backpack/community-forum/assets/34405837/18b0ac76-3d95-406e-b084-9f6c9c99c8fe">

when some of you got this error, you can just change your ``` APP_URL=http://localhost ``` to ``` APP_URL=http://your-virtual-host-here.test ```

then after that, you will be fine.

Cheers 🍻

**Comment by pxpm** [comment]
Thanks @christmex for the report and for the solution. 

I will leave this discussion running as it's probable that more people will report issues about assets, and we can then decide what's worth to add to the docs and what not, or if we need to make something more clear!

Once again thanks @christmex 🙏
